api/envoy/extensions/geoip_providers/common/v3/common.proto

```diff
@@ -61,6 +61,15 @@ message CommonGeoipProviderConfig {
     // Header value will be set to either "true" or "false" depending on the check result.
     string anon_proxy = 9
         [(validate.rules).string = {well_known_regex: HTTP_HEADER_NAME ignore_empty: true}];
+
+    // If set, the header will be used to populate the ISP associated with the IP address.
+    string isp = 10
+        [(validate.rules).string = {well_known_regex: HTTP_HEADER_NAME ignore_empty: true}];
+
+    // If set, the IP address will be checked if it belongs to the ISP named iCloud Private Relay.
+    // Header value will be set to either "true" or "false" depending on the check result.
```

Comment:
It's a bit unclear to me that this is a header name.
Can the comment be clarified to denote that: if this is set, a header with that name will be added and its value will be (as described in the comment)